src/app/pages/network/explorer/account/account-detail/account-transfers/account-transfers.component.ts

```diff
@@ -212,7 +213,8 @@ export class AccountTransfersComponent implements OnChanges, OnDestroy {
     // First check if there is a Transfer available.
     if ((eventOrTransfer as pst.Transfer).hasOwnProperty('amount')) {
       const amounts: eventAmounts = [];
-      amounts.push(['amount', new BN((eventOrTransfer as pst.Transfer).amount)]);
+      const value = BigNumber((eventOrTransfer as pst.Transfer).amount).toFixed(0);
```

Comment:
`new BigNumber`

Reply:
the new it's optional so I think it's redundant to add it 🤔 

Do you prefer to add it?

Reply:
Yes, it would be good to be consistent